CITATION.cff

```diff
@@ -0,0 +1,18 @@
+# YAML 1.2
+---
+authors: 
```

Comment:
Only two authors seem to be listed, yet there are many more contributors. Is this a limitation of the .cff format?

Reply:
Oops! I must have committed my changes before hitting the save button. Yes, there are many more contributors than two. I pushed a version that includes the entire author list as it appears on Zenodo.